### ISSUE TITLE
--version prints to stderr instead of stdout

### ISSUE THREAD

**Issue by kachkaev** [wooorm/unified-args#18]
Hi @wooorm! I'm trying to switch to `remark-cli` in https://github.com/Glavin001/atom-beautify/issues/965 after your [comment](https://github.com/Glavin001/atom-beautify/issues/965#issuecomment-356340663), but am failing to configure remark as an external command due to how version is printed (`atom-beautifier` first checks for the command's version and fails because there is nothing in stdout for `remark`).

```bash
remark --version 
# OUTPUT: remark: 8.0.0, remark-cli: 4.0.0

remark --version 2>/dev/null
# NO OUTPUT, EXPECTED remark: 8.0.0, remark-cli: 4.0.0

remark --version 1>/dev/null
# NO OUTPUT EXPECTED, GOT: remark: 8.0.0, remark-cli: 4.0.0
```

What's the reason for printing `--version` into stderr instead of stdout? Could that be changed?

**Comment by wooorm** [comment]
Hmm, interesting!

- `unified-args` only writes to stderr (such as help text, version, other logs, or errors)
- `unified-engine` only writes the output file, if exactly one is processed, to stdout; the message report is written to stderr

I could change it, but maybe you could detect the version from stderr?

**Comment by kachkaev** [comment]
I guess I can find a workaround for `atom-beautify`, however it still feels that seeing `--version` in stdout is the right thing. Both `node --version` and `npm --version` seem to print to stdout. Same for `docker`, `brew`, `kubectl` and `bash` that I just tested quickly.

If the version is printed to stderr, other `remark-cli` users might get into the same trap as I did today – could not understand why I see null inside js for over an hour :–) I understand that a change would be breaking though and would also screw up your nice distinction between modules. So leaving the design decision to you.

In any case, thanks a lot for this awesome mini-ecosystem of tools, I love how remark is done and what's beneath it ❤️

**Comment by wooorm** [comment]
Alright, I’m all for changing this, but I feel it’s pretty low priority, so I’d like to leave this open for some other stuff to go in a major release, okay?

**Comment by kachkaev** [comment]
Sure. I found a workaround meanwhile :–)

**Comment by wooorm** [comment]
Good good! 👍

**Comment by wooorm** [comment]
Released as [unified-args@6.0.0](https://github.com/unifiedjs/unified-args/releases/tag/6.0.0)!
